_posts/patterns/2016-01-28-alerts.md

```diff
@@ -0,0 +1,275 @@
+---
+layout:         pattern
+title:          Alerts
+date:           2015-01-28 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- alerts
+
+slug:           alerts
+
+url_styles:     "patterns/_alerts.scss"
+
+description:    Examples of alerts in a semi-real world context.
+
+info:           Alerts provide information about...TBD
+---
+
+<h3 class="hd-6 example-set-hd">Error w/ Title</h3>
+<div class="example-set">
+    <div class="alert alert-error">
+        <div class="icon-display">
+            <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+        </div>
+
+        <div class="alert-message">
+            <div class="alert-title">
+                Alert Title!
+            </div>
+
+            <div class="alert-copy">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit
+                in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+                cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+            </div>
+        </div>
+
+        <div class="alert-action">
+            <button type="button" class="btn-neutral btn">Alert Action</button>
```

Comment:
NIT: We should try to populate this with content from one of the real alert cases we identified. It could help suggest length, tone, and general purpose of this element.